src/admin.c

```diff
@@ -882,12 +890,15 @@ static bool admin_show_pools(PgSocket *admin, const char *arg)
 	statlist_for_each(item, &pool_list) {
 		pool = container_of(item, PgPool, head);
 		waiter = first_socket(&pool->waiting_client_list);
+		/* XXX: should maxwait include replication connections? */
```

Comment:
Conceptually it would make sense to include replication connections too, but I also don't think it's super important.